frontend/test/components/Button.unit.spec.js

```diff
@@ -3,7 +3,7 @@ import renderer from 'react-test-renderer';
 
 import { render } from 'enzyme';
 
-import Button from './Button';
+import Button from '../../src/metabase/components/Button';
```

Comment:
Dunno if the webpack namespaces are available here but it'd be nice to use `metabase/components/Button` instead of having to do relative paths.